datavec/datavec-python/src/main/java/org/datavec/python/PythonExecutioner.java

```diff
@@ -62,23 +62,36 @@ private static synchronized void init() {
         numpy._import_array();
     }
 
-    private static synchronized void _exec(String code) {
+    private static synchronized void _exec(String code) throws PythonException{
         log.debug(code);
         log.info("CPython: PyRun_SimpleStringFlag()");
 
         int result = PyRun_SimpleStringFlags(code, null);
         if (result != 0) {
             log.info("CPython: PyErr_Print");
             PyErr_Print();
-            throw new RuntimeException("exec failed");
+            throw new PythonException("exec failed"); // TODO: Surface actual python error here
         }
     }
 
-    public static void setVariable(String varName, PythonObject pythonObject) {
+    public static boolean validateVariableName(String s) {
+        if (s.length() == 0) return false;
+        if (!Character.isJavaIdentifierStart(s.charAt(0))) return false;
+        for (int i = 1; i < s.length(); i++)
+            if (!Character.isJavaIdentifierPart(s.charAt(i)))
+                return false;
+        return true;
+    }
+
+
+    public static void setVariable(String varName, PythonObject pythonObject) throws Exception{
+        if (!validateVariableName(varName)){
```

Comment:
Same thing above use PythonException. We should also explain what the variable is and how it relates to PythonObject.